app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationRouter.kt

```diff
@@ -180,36 +180,37 @@ private fun ConversationRouter(
                         )
                     }
                 )
-                composable(
-                    route = ConversationsNavigationItem.Calls.route,
-                    content = {
-                        CallsScreen(
-                            missedCalls = missedCalls,
-                            callHistory = callHistory,
-                            onCallItemClick = openConversation,
-                            onEditConversationItem = onEditConversationItem,
-                            onScrollPositionProviderChanged = onScrollPositionProviderChanged,
-                            onOpenUserProfile = openProfile,
-                            openConversationNotificationsSettings = onEditNotifications,
-                            onJoinCall = onJoinCall
-                        )
-                    }
-                )
-                composable(
-                    route = ConversationsNavigationItem.Mentions.route,
-                    content = {
-                        MentionScreen(
-                            unreadMentions = unreadMentions,
-                            allMentions = allMentions,
-                            onMentionItemClick = openConversation,
-                            onEditConversationItem = onEditConversationItem,
-                            onScrollPositionProviderChanged = onScrollPositionProviderChanged,
-                            onOpenUserProfile = openProfile,
-                            openConversationNotificationsSettings = onEditNotifications,
-                            onJoinCall = onJoinCall
-                        )
-                    }
-                )
+                // Disable for now until it is supported
```

Comment:
I would suggest to remove the navigation bar in a different PR instead of removing `Missed Calls` and `Mentions`. It make no sense to keep it with one item

Reply:
I mean, we will not be ableto show the funcionality of counting unread conversation if we remove the navigation bar I think I am going to ask it in the gorup chat I am fine with removing it of course ;P